### ISSUE TITLE
Private Docker Registry login

### ISSUE THREAD

**Issue by anthonytec2** [polyaxon/polyaxon#694]
### Describe the bug
2020-03-19 22:05:35 UTC -- Building: Step 1/6 : FROM [PRIVATE_REGISTRY]/[REPO]:tune
2020-03-19 22:05:35 UTC -- Building:
2020-03-19 22:05:35 UTC -- ERROR: Get https://gitlab.saicny.com:4567/v2/saic-ny/robotics/manifests/param_tuning: denied: access forbidden
2020-03-19 22:05:35 UTC -- Traceback (most recent call last):
2020-03-19 22:05:35 UTC --   File "dockerizer/build_cmd.py", line 38, in <module>
2020-03-19 22:05:35 UTC --     nocache=nocache)
2020-03-19 22:05:35 UTC --   File "/usr/local/lib/python3.7/site-packages/polyaxon_dockerizer/builder.py", line 238, in build_and_push
2020-03-19 22:05:35 UTC --     sleep_interval=sleep_interval)
2020-03-19 22:05:35 UTC --   File "/usr/local/lib/python3.7/site-packages/polyaxon_dockerizer/builder.py", line 189, in build
2020-03-19 22:05:35 UTC --     registries=registries)
2020-03-19 22:05:35 UTC --   File "/usr/local/lib/python3.7/site-packages/polyaxon_dockerizer/builder.py", line 167, in _build
2020-03-19 22:05:35 UTC --     raise BuildException('The docker image could not be built.')
2020-03-19 22:05:35 UTC -- polyaxon_dockerizer.exceptions.BuildException: The docker image could not be built.

I have setup a polyaxon.yaml file, and have included a private docker image. I setup the  private registry on polyaxon(Secret+Private Registry) using a .docker/config.json. I have verified outside of polyaxon, I can in a Kubernetes job pull from this registry. Although, whenever I put this in a build job, I receive: denied access forbidden error. What is the best way for me to go about debugging this? 

# Polyaxon.yaml
version: 1
kind: experiment
build:
  image: [PRIVATE_REGISTRY]/[REPO]:tune
  build_steps:
    - ls
run:
  cmd: sleep 1000

### Environment
Cluster info:

--------------  ----------------------------------------
major           1
minor           16
compiler        gc
platform        linux/amd64
build_date      2020-03-12T20:52:22Z
git_commit      ec6eb119b81be488b030e849b9e64fda4caaf33c
go_version      go1.13.8
git_version     v1.16.8
git_tree_state  clean
--------------  ----------------------------------------
  sequence  name      hostname    role    memory       cpu    n_gpus
----------  --------  ----------  ------  ---------  -----  --------
         4  m1  m1    agent   36.39 Gb     6         2
         3  m2  m2    agent   36.39 Gb     6        2
         2  m3  m3    agent   36.39 Gb     6         2
         1  m4  m4    master  36.39 Gb     6         2

kubectl version
Client Version: version.Info{Major:"1", Minor:"15", GitVersion:"v1.15.5", GitCommit:"20c265fef0741dd71a66480e35bd69f18351daea", GitTreeState:"clean", BuildDate:"2019-10-15T19:16:51Z", GoVersion:"go1.12.10", Compiler:"gc", Platform:"darwin/amd64"}
Server Version: version.Info{Major:"1", Minor:"16", GitVersion:"v1.16.8", GitCommit:"ec6eb119b81be488b030e849b9e64fda4caaf33c", GitTreeState:"clean", BuildDate:"2020-03-12T20:52:22Z", GoVersion:"go1.13.8", Compiler:"gc", Platform:"linux/amd64"}

**Comment by mouradmourafiq** [comment]
The logs show the error:

> ERROR: Get https://gitlab.saicny.com:4567/v2/saic-ny/robotics/manifests/param_tuning: denied: access forbidden

You don't have access (or you did not grant access) to that gitlab repo.

**Comment by anthonytec2** [comment]
I understand this error message, I have given the repository access to the registry. I have looked inside the image that is being use to perform the build command and it seems how you get a private registry is through an environment variable. I do not see the repository I have linked on the Polyaxon site in any env variable, what can I do further to look into this? Also, I went through the build code and printed out private registries, of which none are printed.
